lib/candig_server/docker-compose.yml

```diff
@@ -22,12 +23,16 @@ services:
         window: 120s
       labels:
         - traefik.enable=true
-        - traefik.port=${CANDIG_SERVER_PORT}
-        - traefik.docker.network=${DOCKER_NET}
-        - traefik.frontend.rule=Host:candig-server.${CANDIG_DOMAIN}
+        - traefik.port=3001
+        - traefik.docker.network=docker-compose-test-network
+        - traefik.frontend.rule=Host:candig-server.localhost
     logging:
       driver: "fluentd"
       options:
         tag: docker.{{.Name}}
         fluentd-async-connect: "true"
     command: ["--host", "0.0.0.0", "--port", "3000"]
+
+networks:
```

Comment:
this is not needed, i assume you were doing some testing and did not clean up these changes